packages/router/router.ts

```diff
@@ -1244,6 +1273,10 @@ export function createRouter(init: RouterInit): Router {
     let result: DataResult;
     let actionMatch = getTargetMatch(matches, location);
 
+    if (actionMatch.route.lazy) {
+      await loadLazyRouteModules([actionMatch], manifest, request.signal);
+    }
```

Comment:
At each entry point we kick this off _after_ `updateState` so it's part of the `submitting`/`loading` state but before we try to call actions/loaders.

Reply:
And in all cases if we got aborted we should short circuit out